docs/env_vars.md

```diff
@@ -0,0 +1,21 @@
+wlroots reads these environment variables
+
+wlroots specific
+----------------
+* *WLR_DRM_DEVICES*: specifies the DRM device instead of auto probing it
+* *WLR_DRM_NO_ATOMIC*: set to 1 to use legacy DRM interface instead of atomic mode setting
```

Comment:
Technically it's if it's defined at all, not if it's set to 1.

Reply:
code should be fixed